hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -616,6 +651,7 @@ public DeltaSync getDeltaSync() {
           }
         } finally {
           shutdownCompactor(error);
+          shutdownClusteringService(error);
```

Comment:
can we have a single method and call it shutdownAsyncServices or backgroundServices and shut down all such services within that method ?

Reply:
Done.